### PR TITLE
Add Vale instructions.

### PR DESCRIPTION
**Pull request by bgravenorst**
Add instructions on how to use Vale.

### CODE REVIEW

File: docs/contribute/run-vale.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+---
+
+# Run the spelling and style linter
+
+The documentation suite uses [Vale](https://vale.sh/) as the style guide and spelling linter.
+Vale is currently integrated into the continuous integration (CI) pipeline and is executed on each
+pull request (PR) using a GitHub action. You can select the **Details** link to view the failures.
+
+:::info important
+The Vale GitHub workflow runs on all Markdown files in the document, not only the one's you've updated.
```
Comment (jlwllmr) [inline docs/contribute/run-vale.md:13]:
```suggestion
The Vale GitHub workflow runs on all Markdown files in the document, not only the ones you've updated.
```

File: docs/contribute/run-vale.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+   ```bash
+   git clone https://github.com/Consensys/docs-gha.git
+   ```
+
+1. Override the default location of the `.vale.ini` file by setting the `VALE_CONFIG_PATH` environment
+    variable to the location of the file in the repo. For example, on macOS this is:
+
+    ```bash
+    export VALE_CONFIG_PATH="/Users/jdoe/documentation/docs-gha/spelling/.vale.ini"
```
Comment (m4sterbunny) [inline docs/contribute/run-vale.md:43]:
```suggestion
    export VALE_CONFIG_PATH="/Users/{user-name}/documentation/docs-gha/spelling/.vale.ini"
```

Comment (m4sterbunny) [reply docs/contribute/run-vale.md:43]: alt is username like cherry-blossom
Avoid jdoe --> John Doe is the boilerplate name for an unidentified male corpse(!)

File: docs/contribute/run-vale.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+    ![VS Code extension settings](../assets/vs-code-ext.png)
+
+1. Restart VS Code.
+
+## Contribute to the spell checker
+
+You can contribute to the spell checker by submitting a PR to the [`docs-gha` repository](https://github.com/Consensys/docs-gha).
+
+You can add your words to the [`accept.txt`](https://github.com/Consensys/docs-gha/blob/main/spelling/styles/config/vocabularies/Consensys-common/accept.txt) file.
```
Comment (m4sterbunny) [inline docs/contribute/run-vale.md:84]:
```suggestion
You can add your words to the [`accept.txt`](https://github.com/Consensys/docs-gha/blob/main/spelling/styles/config/vocabularies/Consensys-common/accept.txt) file.

```

File: docs/contribute/run-vale.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+---
+
+# Run the spelling and style linter
+
+The documentation suite uses [Vale](https://vale.sh/) as the style guide and spelling linter.
+Vale is currently integrated into the continuous integration (CI) pipeline and is executed on each
+pull request (PR) using a GitHub action. You can select the **Details** link to view the failures.
+
+:::info important
+The Vale GitHub workflow runs on all Markdown files in the document, not only the ones you've updated.
```
Comment (m4sterbunny) [inline docs/contribute/run-vale.md:13]:
```suggestion
While the Vale GitHub workflow runs on all Markdown files, not only the ones you've updated, only items related to your PR receive alerts.
```

File: docs/contribute/run-vale.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+    ![VS Code extension settings](../assets/vs-code-ext.png)
+
+1. Restart VS Code.
+
+## Contribute to the spell checker
+
+You can contribute to the spell checker by submitting a PR to the [`docs-gha` repository](https://github.com/Consensys/docs-gha).
+
+You can add your words to the [`accept.txt`](https://github.com/Consensys/docs-gha/blob/main/spelling/styles/config/vocabularies/Consensys-common/accept.txt) file.
```
Comment (m4sterbunny) [inline docs/contribute/run-vale.md:84]:
```suggestion
Learn more about how to [configure Vale](https://github.com/Consensys/docs-gha/tree/main/spelling#configure-vale).
```

**Review by m4sterbunny — CHANGES_REQUESTED** [review]
LGTM added/amended some items

Consider adding warning -- Vale sucks at error messages, so if you pass it a local file that doesn't exist, instead of saying so, you get:  `0 errors, 0 warnings and 0 suggestions in stdin.`

File: docs/contribute/run-vale.md (hunk 5 above)
Comment (bgravenorst) [reply docs/contribute/run-vale.md:84]: Yes I agree that info can probably be added in the "Configure" section" https://docs-template.consensys.io/configure

Perhaps in a followup PR.

**Pull request merged**